lib/vanagon/component/dsl.rb

```diff
@@ -39,8 +39,8 @@ def _component
       # We only magically handle get_ methods, any other methods just get the
       # standard method_missing treatment.
       #
-      def method_missing(method, *args)
-        attribute_match = method.to_s.match(/get_(.*)/)
+      def method_missing(method_name, *args)
+        attribute_match = method_name.to_s.match(/get_(.*)/)
```

Comment:
Nah, it doesn't really belong any place. it's just that `method` is almost but not quite a reserved word (much like `name`) so we should maybe be more specific.